lib/account.rb

```diff
@@ -0,0 +1,27 @@
+class Account
+  attr_reader :entries
+  attr_reader :balance
+
+  def initialize(entries = [], balance = 2000)
+    @entries = entries
+    @balance = balance
+  end 
+
+  def add_entry(entry)
+    @entries << entry
+  end
+
+  def print_statement
+    puts 'date || credit || debit || balance'
+    @entries.each do |entry|
+      puts "#{entry.date} ||" + " ||" + " #{entry.debit} ||" + " #{balance_update(entry)}" if entry.credit.nil?
+      puts "#{entry.date} ||" + " #{entry.credit} ||" + " ||" + " #{balance_update(entry)}" if entry.debit.nil?
+    end
+  end
+  
+  private
+
+  def balance_update(entry)
```

Comment:
`balance_update` has multiple responsibilities.  Is there a way to refactor it to avoid this?